File: app/src/main/java/me/hachy/routineweek/util/TagColor.java
package me.hachy.routineweek.util;


import java.util.Arrays;

public enum TagColor {
    None("none"),
    Blue("blue"),
    Red("red"),
    Green("green"),
    Yellow("yellow");

    private String name;

    private TagColor(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public static String[] names() {
        return Arrays.toString(TagColor.values()).replaceAll("^.|.$", "").split(", ");
    }
}

File: app/src/main/java/me/hachy/routineweek/util/TagColor.kt
package me.hachy.routineweek.util


import java.util.Arrays

enum class TagColor {
    None, Blue, Red, Green, Yellow;

    companion object {

        fun names(): Array<String> {
            return Arrays.toString(TagColor.values()).replace("^.|.$".toRegex(), "").split(", ".toRegex()).dropLastWhile { it.isEmpty() }.toTypedArray()
        }
    }
}
